indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +86,36 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHEditTrack(RHManageTrackBase):
+    """Edit a track"""
+
+    def _process(self):
+        form = TrackForm(obj=FormDefaults(self.track))
+        if form.validate_on_submit():
+            track = update_track(self.track, form.data)
+            flash(_('Track "{}" has been modified.').format(track.title), 'success')
+            return jsonify_data(flash=False)
+        return jsonify_form(form)
+
+
+class RHSortTracks(RHManageTracksBase):
+    """Sort the list of tracks"""
+
+    def _process(self):
+        sort_order = request.json['sort_order']
+        tracks = {t.id: t for t in self.event_new.tracks}
+        for position, track_id in enumerate(sort_order, 1):
+            if track_id in tracks:
+                tracks[track_id].position = position
+
+
+class RHDeleteTrack(RHManageTrackBase):
+    """Delete a track"""
+
+    def _process(self):
+        delete_track(self.track)
```

Comment:
We should not give the option of deleting in the UI if it's not possible. In the case the request arrives it can be one of two options: 
1. In between rendering the page and clicking the button, an abstract has been accepted in that track.
2. Somebody is messing around.

The easy solution is raising an exception and the user-friendly solution is handling it so that it doesn't look like an error. I would go for the latter.